tests/ethpm/test_package_init_from_registry.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from solc import compile_source
+
+import ethpm
+
+from ethpm import Package
+
+from ethpm.exceptions import UriNotSupportedError
+
+from ethpm.constants import REGISTRY_SOURCE
+
+
+# mock out http req to IPFS gateway
+# `fetch_ipfs_package` returns local 'owned' pkg
+@pytest.fixture(autouse=True)
+def mock_request(monkeypatch, owned_manifest):
```

Comment:
Just leaving a note that we should be able to drop this mocking once the IPFS fetching is done with a backend system, as we can just swap in our own dummy backend that just returns the content from memory or something like that.

Reply:
sounds good - added it to the ipfs issue